File: submissions/Becrowd/_1176(3).c
#include <stdio.h>


int fibo(int x){
    if (x == 0){
        return 0;
    }

    if (x == 1){
        return 1;
    }

    int i;
    int previous = 1;
    int now = 1;
    int temp = 0;
    for (i = 0; i < x -2; i++){
        temp = now;
        now = previous + now;
        previous = temp;
    }
    return now;
}

int main(){
    int N;
    scanf("%i", &N);

    int i;
    int x;
    for (i = 0; i < N; i++){
        scanf("%i", &x);
        printf("Fib(%i) = %i\n", x, fibo(x));
    }
}

File: submissions/Becrowd/_1176(3).py
# #1176 Fibonacci Array
def fibo(x):
    if x == 0:
        return 0
    elif x == 1:
        return 1
    
    now = 1
    previous = 1
    for i in range(0, x-2):
        now = previous + now
        previous = now - previous 
    return now

N = int(input())
for i in range(0, N):
    x = int(input())
    print(f"Fib({x}) = {fibo(x)}")

